grovedb/src/operations/aux.rs

```diff
@@ -0,0 +1,17 @@
+use storage::Storage;
+
+use crate::{Error, GroveDb};
+
+impl GroveDb {
+    pub fn put_aux(&mut self, key: &[u8], value: &[u8]) -> Result<(), Error> {
+        Ok(self.meta_storage.put_aux(key, value)?)
+    }
+
+    pub fn delete_aux(&mut self, key: &[u8]) -> Result<(), Error> {
+        Ok(self.meta_storage.delete_aux(key)?)
+    }
+
+    pub fn get_aux(&mut self, key: &[u8]) -> Result<Option<Vec<u8>>, Error> {
+        Ok(self.meta_storage.get_aux(key)?)
```

Comment:
Does it stores with prefixes in the same CF like merk data

Reply:
No, this storage has empty prefix and it all be stored as is in aux column family

Reply:
Sounds good! Thank you!